4-projects/modules/base_env/example_peering_project.tf

```diff
@@ -261,3 +234,110 @@ resource "google_compute_firewall" "allow_lb" {
 
   target_tags = ["allow-lb"]
 }
+
+// Allow SSH via IAP when using the Firewall Secure Tags.
```

Comment:
Wouldn't that be SSH + RDP?
I ask because looks like at the code we are removing, we have more details related to where was the SSH and RDP code flow (also details related to which SO uses which protocol)